group-coordinator/src/test/java/org/apache/kafka/coordinator/group/runtime/CoordinatorRuntimeTest.java

```diff
@@ -166,9 +214,27 @@ public CoordinatorBuilder<MockCoordinator, String> withLogContext(
             return this;
         }
 
+        @Override
+        public CoordinatorBuilder<MockCoordinator, String> withTime(
+            Time time
+        ) {
+            return this;
```

Comment:
Do we just ignore time here and we are required to override this method with a no-op?

Reply:
Right.